sirepo/package_data/static/js/sirepo.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 SIREPO.srlog = console.log.bind(console);
 SIREPO.srdbg = console.log.bind(console);
+SIREPO.traceWS = true;
```

Comment:
I don't think we want this to default to true - it junks up the console on dev.

Reply:
Whoops 6f1ddeed712cd546c6ade5d32ed14fdcb873db7d

Reply:
Doesn't look like you pushed your changes @robnagler

Reply:
@rorour thanks I had a messed up git config and emacs didn't tell me...